<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="holder">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>821</width>
      <height>951</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="input_text_widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>741</width>
      <height>941</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>501</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="inputLayout">
      <item>
       <widget class="QTextEdit" name="textEdit_input"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="load_file_btn">
          <property name="text">
           <string>Загрузить из файла</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="solve_text_btn">
          <property name="text">
           <string>Вычислить функцию</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Введите координаты точек в 2 стобца X и Y через пробел</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="res_widget" native="true">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>9</y>
      <width>731</width>
      <height>941</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>9</y>
       <width>681</width>
       <height>931</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="res_layout"/>
    </widget>
   </widget>
   <widget class="QWidget" name="start_widget" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>791</width>
      <height>931</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_text_input">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>331</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Текст / файл</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_func_input">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>410</y>
       <width>331</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Функция</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="function_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>771</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="formulas_layout"/>
    </widget>
    <widget class="QPushButton" name="select_func_btn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>380</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вычислить функцию</string>
     </property>
    </widget>
    <widget class="QComboBox" name="select_func_comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>320</y>
       <width>731</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Начало:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spin_start">
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Конец</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spin_end">
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Количество</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spin_count">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>holder</zorder>
   <zorder>input_text_widget</zorder>
   <zorder>start_widget</zorder>
   <zorder>res_widget</zorder>
   <zorder>function_widget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
